### ISSUE TITLE
Docstring not formatted correctly

### ISSUE THREAD

**Issue by sdc50** [tethysplatform/tethys_dataset_services#19]
Sphinx is warning about the formatting of this doc string block:

https://github.com/tethysplatform/tethys_dataset_services/blob/master/tethys_dataset_services/engines/geoserver_engine.py#L1251-L1261

I'm not sure what is wrong, but it may be that `Args` doesn't have a `:`.

Sphinx Warning:

```
.../python3.7/site-packages/tethys_dataset_services/engines/geoserver_engine.py:docstring of tethys_dataset_services.engines.GeoServerSpatialDatasetEngine.create_sql_view:31: WARNING: Definition list ends without a blank line; unexpected unindent.
```

**Comment by msouff** [comment]
I think the problem is this: https://github.com/sphinx-doc/sphinx/issues/4479

The noqa we have at line 1260 supports that theory: https://www.flake8rules.com/rules/W605.html

The solution would be to add r in front of line 1248 (r""") or individually in front of the strings with backslashes on line 1260.

Can you maybe try that and see if it does anything?

**Comment by swainn** [comment]
Fixed in commit [68291b8](https://github.com/tethysplatform/tethys_dataset_services/commit/68291b8f1e972c54c2c0ac006f8291202f40f619)
